src/components/ToolTip.svelte

```diff
@@ -0,0 +1,43 @@
+<script>
+    export let title = '';
```

Comment:
"title" is a strange description for what is being shown, I think "description" or "details" might be better

Reply:
I'll be more generic, something like `text`?